js_modules/dagit/packages/app/package.json

```diff
@@ -45,11 +45,11 @@
     "css-loader": "4.3.0",
     "dotenv": "8.2.0",
     "dotenv-expand": "5.1.0",
-    "eslint": "^7.23.0",
+    "eslint": "7.32.0",
```

Comment:
Big fan of locking this to a specific version, I hate when point releases fix rules and break the build 😅